pages/news_item/news_item.ejs

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <%- include('../partials/head'); %>
+    <link href="/stylesheets/issues.css" rel="stylesheet" />
+  </head>
+  <body>
+    <%- include('../partials/navbar', {navPage: 'news_item'}); %>
+    <div id="content" class="container">
+      <%- include('../partials/navtabs'); %>
+
+      <a class="btn btn-primary mb-4" href="<%= urlPrefix %>/news_items" role="button">
+        <i class="fa fa-arrow-left" aria-hidden="true"></i>
+        Back to news_items
```

Comment:
Maybe something human-readable like `Back to list of news items`, `Back to news items`, or whatever - anything other than `news_items`.

Reply:
Oops, good catch, this was a search-and-replace failure. Fixed.